src/MessagingThreadHistory/MessagingThreadHistory.tsx

```diff
@@ -127,8 +131,11 @@ function _interleaveMessagesWithDividers(
       const messageDay = _formatDateForDivider(message.timestamp);
       if (currentDay !== messageDay) {
         messagesWithDividers.push(
-          <div key={`divider-${messageDay}`} className={cssClasses.DIVIDER}>
-            {messageDay}
+          <div role="row">
+            <div role="gridcell" key={`divider-${messageDay}`} className={cssClasses.DIVIDER}>
+              {messageDay}
+            </div>
+            ,
```

Comment:
FOUND IT!!!!

Reply:
rogue comma

Reply:
who do you `git blame`? probably me.

Reply:
I actually blame git for not being accessible enough for a person with good enough sight to catch this! It's all of our faults 😞